Fixing port related description (#16819)
kubernetes/website: content/en/docs/tutorials/hello-minikube.md

4. Katacoda environment only: Click the plus sign, and then click **Select port to view on Host 1**.

> 5. Katacoda environment only: Note 5 digit port number displayed opposite to `8080` in services output. This port number generates        randomly and it can be different for you. In this case, the port number is `30369`. So Type `30369` in port number text box appearing on Katacoda web page, and then click.

Reviewer: ```suggestion
5. Katacoda environment only: Note the 5 digit port number displayed opposite to `8080` in services output. This port number is randomly generated and it can be different for you. Type `30369` in the port number text box, then click Display Port.
```

Author: Made changes as per review comments
